sympy/solvers/tests/test_ode.py

```diff
@@ -2901,3 +2901,33 @@ def test_issue_14395():
     sol = Eq(f(x), (C1 - x/3 + sin(2*x)/3)*sin(3*x) + (C2 + log(cos(x))
         - 2*log(cos(x)**2)/3 + 2*cos(x)**2/3)*cos(3*x))
     assert dsolve(Derivative(f(x), x, x) + 9*f(x) - sec(x), f(x)) == sol
+
+
+def test_sysode_linear_neq_order1():
+    
+    from sympy import Function
+    from sympy import symbols
+    from sympy import dsolve
```

Comment:
The imports for `Function`, `symbols`, `dsolve` are not required, as they are already imported globally in the file.
Th blank line after `def test_sysode_linear_neq_order1():` can also be removed.